sdcm/nemesis.py

```diff
@@ -2010,8 +2010,8 @@ def get_ks_with_few_tables(keyspace_table):
         self.log.debug(result)
 
     # NOTE: '2022.1.rc0' is set in advance, not guaranteed to match when appears
-    # NOTE: current variant of the toppartitions feature is supported since 4.5 OSS
-    @scylla_versions(("4.5.rc1", None), ("2022.1.rc0", None))
+    # NOTE: current variant of the toppartitions feature is supported since 4.6 OSS
+    @scylla_versions(("4.6.rc0", None), ("2022.1.rc0", None))
```

Comment:
@roydahan , @bentsi , other way to takkle this problem would be to fix `top partitions class` to take into consideration scylla_version when it is generating a command

Reply:
Yes, that would be the better solution.
I think this one was a quick and easy fix.

Reply:
So, what shall we do with it? Merge or/and add proper fix (create task for that)?